# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1315079018 25200
#      Sat Sep 03 12:43:38 2011 -0700
# Node ID a3d21c3c0f585c7f32b996c2849edca4fbecaa50
# Parent  a0d854f079d239909d1b9830ca529e7dcabf575e
codesprint: No tests needed for alias of tmpnam function.

* tempname.m: Mark file as tested with assert (1).

diff --git a/scripts/miscellaneous/tempname.m b/scripts/miscellaneous/tempname.m
--- a/scripts/miscellaneous/tempname.m
+++ b/scripts/miscellaneous/tempname.m
@@ -24,8 +24,12 @@
 ## @seealso{tmpnam}
 ## @end deftypefn
 
 function filename = tempname (varargin)
 
   filename = tmpnam (varargin{:});
 
 endfunction
+
+
+%% No tests needed for alias.
+%!assert (1)
